### ISSUE TITLE
Tutorials and Walkthrough

### ISSUE THREAD

**Issue by JacopoPan** [microsoft/AirSim#2991]
I tried tor write a couple _extremely_ simple, self-contained tutorials on

- [Using AirSim's APIs / Compiling UE4 and AirSim](https://github.com/JacopoPan/a-minimalist-guide/blob/master/Part3-Using-AirSim.md)
- [Modifying/recompiling AirSim's Unreal plugin and using custom UE4 projects and meshes](https://github.com/JacopoPan/a-minimalist-guide/blob/master/Part4-Modifying-AirSim.md)

If anyone wants to check them out/correct me on anything I did wrong 

I was motivated in part by the fact that I found the documentation on https://microsoft.github.io/AirSim/ to be a bit scattered but, more importantly, some of it is as old as 5 years, while some was updated as recently last month: **is there any plan to re-factor/simplify/make current the existing body of documentation?**

**Comment by jonyMarino** [comment]
@JacopoPan excellent initiative! We are very interested in these types of contributions. I will be looking at it and coming back to you soon.

**Comment by JacopoPan** [comment]
@jonyMarino I was happy to see this had positive feedback: if you think there are certain parts I should/could polish and make into a PR, please let me know

**Comment by jonyMarino** [comment]
@JacopoPan It seems to us that this documentation corresponds to different sections of the current documentation. We will see it well and return to you

**Comment by JacopoPan** [comment]
Yes, of course: there are some additional terminal snippets, information I learned from open and closed Issues, and the custom pawn instructions and RLlib integration are new) but, as I wrote in the first message, one of the (I believe) limits of the current documentation is how scattered across pages it currently is (and that's what I think it would be useful to address)

**Comment by jonyMarino** [comment]
@JacopoPan Sorry for the delays in incorporating your contribution. We are making efforts so that the contributions of the community are incorporated more quickly. Could you make a pull request for this and we continue there? It will get more attention if you do that
